packages/common/src/router-v2.js

```diff
@@ -0,0 +1,86 @@
+const { getRoutePaths } = require('./router');
+const { Router } = require('express');
+
+/**
+ * @typedef {import('express').IRouter} IRouter
+ * @typedef {import('express').Handler} Handler
+ * @typedef {Object<string, IRouter>} RouteDict
+ * @typedef {import('./router-v2-types').HttpMethods} HttpMethods
+ * @typedef {import('./router-v2-types').RouterModule} RouterModule
+ * @typedef {{ includeRoot?: boolean, backwardsCompatibilityModeEnabled?: boolean }} Options
```

Comment:
would be useful to have docs explaining what the options do

Reply:
added as part of the readme

Reply:
would have been nice in jsdoc but all good